### PR TITLE
[bounty] Add "BCHN lead maintainer report 2020-03-14" article

### PR DESCRIPTION
**Pull request by merc1er**
#66

### CODE REVIEW

File: blog/lead-maintainer-report-march-2020.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+*   pool.bitcoin.com
+
+*   ASICseer
+
+*   SBI Crypto
+
+Several sites have sprung up to monitor adoption of `/BCHN/` signaling miners / pools:
+
+*   [https://bitcoincash.network/voting/](https://bitcoincash.network/voting/) (at time of writing, 6.9% over last 2016 blocks in period 624961 - 626976)
```
Comment (ftrader) [inline blog/lead-maintainer-report-march-2020.md:27]: In the rendered HTML, the space before `(at the time` is dropped which looks bad

File: blog/lead-maintainer-report-march-2020.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+*   [https://cash.coin.dance/blocks](https://cash.coin.dance/blocks) has added a diagram over last 1000 blocks, it is at 9,3% at time of writing
```
Comment (ftrader) [inline blog/lead-maintainer-report-march-2020.md:29]: In the rendered HTML, the space before `(has added` is dropped which looks bad

**Review by ftrader — CHANGES_REQUESTED** [review]
Minor layout issues which seem to be a product of Markdown -> HTML conversion

Would be good if we know how to fix / work around these, I suspect they will come up again and again where such URLs and normal text mix (maybe only in lists)

File: blog/lead-maintainer-report-march-2020.md (hunk 1 above)
Comment (merc1er) [reply blog/lead-maintainer-report-march-2020.md]: Weird. I removed the "https" prefix from the link and it worked. Prob some issue with the `techy` dependency.

It should be fixed now.

**Comment by ftrader** [comment]
Thanks for fixing the spacing issue.

**Review by ftrader — APPROVED** [review]

**Pull request merged**